### DISCUSSION TITLE
webrtc not working

### DISCUSSION THREAD

**Discussion by sharmasulekh87** [bluenviron/mediamtx#2696] [Questions]
## Which version are you using?

v1.1.1

## Which operating system are you using?

- [ ] Linux amd64 standard
- [ ] Linux amd64 Docker
- [ ] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [ ] Linux arm6 Docker
- [X] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [ ] macOS amd64 Docker
- [ ] Other (please describe)

## Describe the issue

I have setup the rtsp server on Windows Server 2022 with multiple cameras. On local network only one camera webrtc is working in browser. Also with NAT forwarding , none of the camera webrtc is working; For all cases, it shows deadline exceeded close connection error.

## Describe how to replicate the issue

1. start the server
2. publish with multiple camera rtsp source
3. read with web browser

## Did you attach the server logs?

yes 

![image](https://github.com/bluenviron/mediamtx/assets/78798753/6d52bdc2-f428-439d-916c-c96ba1f738a6)

![image](https://github.com/bluenviron/mediamtx/assets/78798753/77b4d18c-713a-474c-992c-faf03a187160)

## Did you attach a network dump?

no

**Comment by bdariusb** [comment]
please provide the settings you use. In the logs there are a lot of  `path 'test' is not configured` entries, that sounds like wrong configuration or you are calling the wrong path

**Comment by aler9** [comment]
Hello, in MediaMTX 1.3.0, the WebRTC connectivity mechanism was rewritten in order to make connectivity simpler. Just fill `webrtcAdditionalHosts` with all your private and public IPs. Instructions are here: https://github.com/bluenviron/mediamtx#connectivity-issues

[a comment by an automated account is not shown]
